src/views/AmbassadorsHub.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>This is the ambassadors hub. Try viewing an ambassador <a href="/ambassador/1">here</a></div>
```

Comment:
To keep vue-router informed, the `<a />` can be `<router-link to="/ambassador/1">here</router-link>` instead